implementations/rust/ockam/ockam_api/src/authenticator/enrollment_tokens/acceptor.rs

```diff
@@ -27,6 +27,7 @@ impl EnrollmentTokenAcceptor {
         Self { tokens, members }
     }
 
+    #[instrument(skip_all, fields(otc = otc.to_string(), from = %from))]
```

Comment:
OTC is a sensitive field, if the user is already enrolled, the OTC code won't be consumed.
Maybe we can track the hash of the OTC?

Reply:
Done.